src/presentation/controller/registration/LimitedPartnershipController.ts

```diff
@@ -110,6 +110,30 @@ class LimitedPartnershipController extends AbstractController {
     };
   }
 
+  closeTransaction(): RequestHandler {
+    return async (request: Request, response: Response, next: NextFunction) => {
+      try {
+        const { tokens, ids } = super.extract(request);
+
+        // TODO Use the response from this call to get hold of the payment URL, when the
+        //      payment journey is implemented
+        //
+        //      E.g.   apiResponse.headers?.["x-payment-required"];
+        await this.limitedPartnershipService.closeTransaction(
+          tokens,
+          ids.transactionId
+        );
+
+        const pageType = super.extractPageTypeOrThrowError(request, RegistrationPageType);
+        const pageRouting = super.getRouting(registrationsRouting, pageType, request);
+
+        response.redirect(pageRouting.nextUrl);
```

Comment:
The registration POST routing for ‘check your answers’ used to call the `redirectAndCacheSelection()` function but I don’t think that’s required (not sure why it was done), so I’ve changed the code to just do a redirect now after closing the transaction. Confirmation page seems to display fine.